src/index.js

```diff
@@ -20,3 +21,20 @@ export {default as UsageJsonFormatter} from './formatter/usage_json_formatter'
 export const defineSupportCode = SupportCodeFns.add
 export const getSupportCodeFns = SupportCodeFns.get
 export const clearSupportCodeFns = SupportCodeFns.reset
+
+const cwd = process.cwd()
+const step = helpers.defineStepFactory(cwd)
+const hook = helpers.defineHookFactory(cwd)
+export const defineStep = (pattern, options, code) => {
+  SupportCodeFns.addStepDefinition(step(pattern, options, code))
+}
```

Comment:
I expected it to look something like
```js
export function defineStep(args...) {
  defineSupportCode(function(methods) {
    methods.defineStep(args...)
  })
}
```
I think that prevents the need for any of the extra logic that you added to the support code library